docs/parallel_config.md

```diff
@@ -207,3 +207,6 @@ Windows you will need to specify the whole path to `plantcv-workflow.py`.
 ```shell
 python %CONDA_PREFIX%/Scripts/plantcv-workflow.py --config my_config.json
 ```
+
+**Source Code:** [Here](https://github.com/danforthcenter/plantcv/blob/811e6558a5b4fbf7e5781a9d92834a0b19642bb2/plantcv/parallel/__init__.py)
```

Comment:
This URL looks like it's to a commit rather than a branch, I assume that's not what was intended?

Reply:
Oh true. I can fix that!